assets/dev/js/editor/controls/dimensions.js

```diff
@@ -92,27 +90,10 @@ ControlDimensionsItemView = ControlBaseUnitsItemView.extend( {
 	},
 
 	fillEmptyDimensions: function() {
-		var dimensions = this.getPossibleDimensions(),
```

Comment:
If you remove this functionality you should also remove the entire linked/unlinked functionality and markup